src/main/java/com/twilio/Twilio.java

```diff
@@ -154,19 +151,6 @@ public static synchronized void setEdge(final String edge) {
         Twilio.edge = edge;
     }
 
-    /**
-     * Set the logger configuration file path.
-     *
-     * @param filePath path to logging configuration file
-     * @param loggerContext defaults to false to get the appropriate logger context for the caller.
-     */
-    public static synchronized void setLoggerConfiguration(final String filePath, final boolean... loggerContext) {
```

Comment:
Since we are removing this function, we need to document how to configure the logging. (see comment in the `README.md`)